validator/lib/validateTileset.js

```diff
@@ -32,6 +32,19 @@ function validateNode(root, parent, resolve) {
         var tile = node.node;
         var parent = node.parent;
 
+        if (defined(tile.content.boundingVolume)) {
+            var region = tile.content.boundingVolume.region;
+            var parentRegion = parent.content.boundingVolume.region;
```

Comment:
<img width="581" alt="bv" src="https://cloud.githubusercontent.com/assets/915398/20309783/5aa081a6-ab17-11e6-8a1f-0dadaa2b4d2d.PNG">

In this screenshot with a parent and four children, `parent.boundingVolume` is white, `parent.content.boundingVolume` is blue, `tile.boundingVolume` are red, and `tile.content.boundingVolume` don't exist.